input/size.go

```diff
@@ -168,6 +165,32 @@ const (
 	// WitnessCommitmentTxWeight 224 weight
 	WitnessCommitmentTxWeight = WitnessHeaderSize + WitnessSize
 
+	// BaseAnchorCommitmentTxSize 223 + 43 * num-htlc-outputs bytes
+	//	- Version: 4 bytes
+	//	- WitnessHeader <---- part of the witness data
+	//	- CountTxIn: 1 byte
+	//	- TxIn: 41 bytes
+	//		FundingInput
+	//	- CountTxOut: 1 byte
```

Comment:
Can’t this be >1 byte?

Reply:
I don't see how this is different than the current commitment type, where it is 1 byte? Since there are just 2 extra outputs

Reply:
Good observation, it looks like the spec calculations are not accounting for that: https://github.com/lightningnetwork/lightning-rfc/blob/master/03-transactions.md#expected-weight-of-the-commitment-transaction

It's an existing issue, but it could possibly be worse in the cases where we are targeting a min-relay feerate, as we could be underestimating. We change this constant to target a worst-case nunmber of outputs, but that would lead to overpaying again in most cases. I think the best solution would be to rely less on this base constant, and dynamically calculate the commit size as we create it.